pkg/config/config.go

```diff
@@ -119,3 +119,19 @@ func WriteConfig(key string, config interface{}) error {
 
 	return nil
 }
+
+// LoadConfigFromFile loads an entire yaml file into a map[string]any
+func LoadConfigFromFile(filePath string) map[string]any {
```

Comment:
is the idea to have a separate file from CATTLE_TEST_CONFIG? 

in other words, why can't we just use the existing 'LoadConfig'?

Reply:
maybe providing a readme of what configs will look like would help?

Reply:
Great question, so, Sam you don't want the second unmarshal happening? Let's leave it this way, later we can update `LoadConfig's` first part with similar to this function.